techniques/system/server-roles/1.0/postgres-check.st

```diff
@@ -1,69 +0,0 @@
-#####################################################################################
-# Copyright 2014 Normation SAS
-#####################################################################################
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, Version 3.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#####################################################################################
-
-bundle agent root_postgres_check
-{
-
-  vars:
-
-    SuSE::
```

Comment:
Wasn't this an exception for SLES?

Reply:
this is done here, on all rpm systems
https://github.com/Normation/rudder-packages/blob/branches/rudder/4.1/rudder-reports/SPECS/rudder-reports.spec#L163

Reply:
but do you know why there was an exception for SLES?

Reply:
it's from the old time https://issues.rudder.io/issues/2207